src/server.js

```diff
@@ -1,38 +1,40 @@
-const app = require("./app");
-const mongoose = require("mongoose");
+const mongoose = require('mongoose');
+const server = require('./app').server;
```

Comment:
Would this work `const {server} = require('./app')` or would it mess up other things?

Reply:
yes, this works, and so does `const {io} = require('./app')` in `socket.js`. Thanks!